src/app/ui/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,84 @@
+import { forwardRef } from 'react';
+
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { css } from 'leather-styles/css';
+
+import { ChevronDownIcon } from '@app/ui/icons';
+
+const accordionTriggerStyles = css({
+  position: 'relative',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  py: 'space.04',
+  textStyle: 'label.02',
+  transition: 'background-color 0.2s',
+  "&[data-state='open']": {
+    '& > .accordion-icon': {
+      transform: 'rotate(180deg)',
+    },
+  },
+  '& > .accordion-icon': {
+    transition: 'transform 0.2s',
+  },
+
+  '&::after': {
+    transition: 'all 0.2s',
+    content: '""',
+    position: 'absolute',
+    left: '-2%',
+    right: '-2%',
```

Comment:
Should we use negative spacing units, rather than percentage here?

Reply:
I wonder also if we can reuse any styles from `<Pressable />`, which does the same thing?

Reply:
removed it, due to @fabric-8 comment above